packages/contracts/src/systems/InventorySystem.sol

```
@@ -23,11 +25,42 @@ contract InventorySystem is System {
                     Player.pushObjectIds(CurrentPlayerId.get(), objIds[i]);
                     objIds[i] = 0;
                     RoomStore.setObjectIds(rId, objIds);
+                    itemPickedUp = true;
                     break;
                 }
             }
         }
 
+        if(itemPickedUp == false) {
+        Output.set("That isn't here");
```

I'm not sure its a great idea to use Output in these systems when its an error, we should probably return an error type and handle this externally then we can change the way we display the outputs to the user... this is getting a bit coupled, that said...